src/app.types.tsx

```diff
@@ -38,36 +38,31 @@ export interface CatalogueCategory {
   is_leaf: boolean;
   catalogue_item_properties?: CatalogueCategoryFormData[];
 }
+
 export interface AddManufacturer {
   name: string;
   url?: string;
-  address: Address | undefined;
-  telephone?: string;
-}
-
-export interface AddManufacturerResponse {
-  id: string;
-  name: string;
-  url: string;
-  address: Address;
-  telephone: string;
-  code: string;
+  address: AddAddress;
+  telephone?: string | null;
 }
 
 export interface EditManufacturer {
   name?: string;
   url?: string;
   address?: EditAddress;
-  telephone?: string;
-  id?: string;
+  telephone?: string | null;
+  id?: string | null;
 }
 
-export interface Manufacturer {
-  id?: string;
+export interface ManufacturerDetails {
   name: string;
   url?: string;
```

Comment:
```suggestion
  url?: string | null;
```
In fact testing it, it was null in my case so the type needs to be changed to this regardless.